<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_correlation_bin_settings</class>
 <widget class="QDialog" name="widget_correlation_bin_settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Correlation plot bin settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>281</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_bin_max">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X max</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_time_unit_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>t.u.</string>
   </property>
  </widget>
  <widget class="QwtCounter" name="cnt_bin_low">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999999999999999929757289024535551219930759168.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000000000042420637374017961984.000000000000000</double>
   </property>
   <property name="numButtons">
    <number>0</number>
   </property>
  </widget>
  <widget class="QListWidget" name="list_chans">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_time_unit_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ns</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_bin_min">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X min</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_leftchan">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Left channel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_num_bins">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bin size</string>
   </property>
  </widget>
  <widget class="QwtCounter" name="cnt_bin_high">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999999999999999929757289024535551219930759168.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000000000042420637374017961984.000000000000000</double>
   </property>
   <property name="numButtons">
    <number>0</number>
   </property>
  </widget>
  <widget class="QwtCounter" name="cnt_num_bins">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000000000042420637374017961984.000000000000000</double>
   </property>
   <property name="numButtons">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_time_unit_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ns</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list_chans_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_leftchan_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Right channel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_FCS">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>FCS (log bins)</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtCounter</class>
   <extends>QWidget</extends>
   <header>qwt_counter.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cnt_bin_low</tabstop>
  <tabstop>cnt_bin_high</tabstop>
  <tabstop>cnt_num_bins</tabstop>
  <tabstop>list_chans</tabstop>
  <tabstop>list_chans_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>widget_correlation_bin_settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>widget_correlation_bin_settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
